tests/acs/test_wfc_darktime.py

```diff
@@ -0,0 +1,74 @@
+import subprocess
+import pytest
+
+from ..helpers import BaseACS
+
+
+class TestDarktimeSingle(BaseACS):
+    """
+    The updated darktime correction (March 2020) is applied to data
+    according to the following categories:
+    Full-frame, flashed and unflashed, pre-SM4
+    Full-frame, flashed and unflashed, post-SM4
+    Subarrays, flashed and unflashed, pre-Cycle 24
+    Subarrays, flashed and unflashed, post-Cycle 24
+    """
+    detector = 'wfc'
+    ignore_keywords = ['filename', 'date', 'iraf-tlm', 'fitsdate',
+                       'opus_ver', 'cal_ver', 'proctime', 'history',
+                       'bitpix', 'naxis', 'extend', 'simple']
+
+    def _single_raw_cte(self, rootname):
```

Comment:
I don't think CTE is necessary to test DARKTIME. And it takes too long.

Reply:
Of course - DOH.

Reply:
I know that we've run into issues in the past with the dark subtraction of non-CTE v. CTE corrected data (see e.g. #217). So while this make take a long time to run, I think it's a good idea in the end. Where are these tests executed and are we in danger of timing out on the platform if we leave the CTE correction in?

Reply:
We do need to be mindful of the time it takes to run the as the whole set of HSTCAL tests gets kicked off on the PR.  In the past I have added boundary pushing tests which took a lot of time (will not delete until I have time to evaluate) to the SKIPPED category so they are not run under normal circumstances.  I used these particular datasets to test for this issue so I could compare against Nate's output as a baseline for "correctness".  I just checked the time it took to run these tests to previous Jenkins runs which did not include these tests, and the processing time is just about the  same (?!).  I will leave the tests as is for now , and I can adjust as necessary.

Reply:
If CTE needs to be tested, at least on a 512 subarray? If this is already a small subarray, then maybe it is okay.

Reply:
I like the idea of using a subarray for this, but to test all the possible combinations for the commanding overheads, then full frame data will be required. Also, I think leaving the test as is and adding it to the SKIPPED category is fine.